test_scripts/API/PerformAudioPassThru/025_ATF_P_PerformAudioPassThru_UI_SUCCESS_TTS_success_false_true.lua

```diff
@@ -0,0 +1,267 @@
+-- In case SDL transfers Speak_request + UI.PerfromAudioPassThru_request to HMI
+-- and receives Speak(<errorCode>) or Speak (resultCode, success:true) response from HMI
+-- and UI.PerformAudioPassThru (SUCCESS) from HMI
+-- SDL must respond PerformAudioPassThru (WARNINGS, success:true) to mobile app
+--
+-- 1. Used preconditions
+-- 1.1. Structure containing all HMI result_codes, success: false + success: true (without SUCCESS itself) is created
+-- 1.2. PerformAudioPassThru RPC is allowed by policy
+-- 1.3. All params used in PerformAudioPassThru_request are present and within bounds
+-- 1.4. AudioPassThruIcon exists at apps sub-directory of AppStorageFolder (value from ini file)
+--
+-- 2. Performed steps
+-- Send PerformAudioPassThru (audioPassThruIcon, other params)
+-- HMI sends to SDL UI.PerformAudioPassThru (SUCCESS)
+-- HMI sends to SDL TTS.Speak (resultcode: HMI_result_code) 
+--
+-- Expected result:
+-- SDL sends UI.PerformAudioPassThru (audioPassThruIcon, other params) to HMI
+-- SDL sends  to HMI UI.PerformAudioPassThru
+-- SDL sends to mobile  TTS.Speak(resultcode: HMI_result_code, success:false/success:true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPostconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+
+--[[ Local variables ]]
+-- info parameter is not specified in scope of the CRQ, it is intended for any applicable future use.
+local hmi_result_code_false = {
+	{ result_code = "UNSUPPORTED_REQUEST", info = "" },
+	{ result_code = "DISALLOWED", info = "" },
+	{ result_code = "USER_DISALLOWED", info = "" },
+	{ result_code = "REJECTED", info = "" },
+	{ result_code = "ABORTED", info = "" },
+	{ result_code = "IGNORED", info = "" },
+	{ result_code = "IN_USE", info = "" },
+	{ result_code = "VEHICLE_DATA_NOT_AVAILABLE", info = "" },
+	{ result_code = "TIMED_OUT", info = "" },
+	{ result_code = "INVALID_DATA", info = "" },
+	{ result_code = "CHAR_LIMIT_EXCEEDED", info = "" },
+	{ result_code = "INVALID_ID", info = "" },
+	{ result_code = "DUPLICATE_NAME", info = "" },
+	{ result_code = "APPLICATION_NOT_REGISTERED", info = "" },
+	{ result_code = "OUT_OF_MEMORY", info = "" },
+	{ result_code = "TOO_MANY_PENDING_REQUESTS", info = "" },
+	{ result_code = "GENERIC_ERROR", info = "" },
+	{ result_code = "TRUNCATED_DATA", info = "" }
+	}
+
+local hmi_result_code_true = {
+	{ result_code_true = "SUCCESS", info = "" },
```

Comment:
@maria-mih, result code SUCCESS should not be included in the test, because when UI and TTS gets SUCCESS, expected mob response should be also SUCCESS.